.github/workflows/publish.yml

```diff
@@ -37,3 +37,95 @@ jobs:
         run: npm publish --tag ${{ steps.npm_tag.outputs.npm_tag }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      - name: Wait for 300 seconds
```

Comment:
instead of hard-coding a time like this, can we just actively check whether the version is available every minute?

Reply:
Yes, we can do it. I will this step to the new job.